cmd/endpoint.go

```diff
@@ -43,7 +44,7 @@ const (
 
 // Endpoint - any type of endpoint.
 type Endpoint struct {
-	*url.URL
+	*url2.URL
```

Comment:
Why not use `url.URL`  and name the `url `instance as `gourl` ?

Reply:
can be done :+1: